src/hooks/useEthBridges.js

```diff
@@ -0,0 +1,39 @@
+import { useCall, useEthers, useToken, useTokenBalance } from "@usedapp/core";
+import { Contract } from "ethers";
+import BridgeABI from "../assets/abis/Bridge.json";
+
+export const ASSETS = ["LLM", "LLD"];
+const ADDRESSES = {
+    LLM: process.env.REACT_APP_LLM_BRIDGE,
+    LLD: process.env.REACT_APP_LLD_BRIDGE,
+};
+
+function useEthBridge(asset) {
+    // FIXME pass this through redux?
+    const { account } = useEthers();
+    const contract = asset && new Contract(ADDRESSES[asset], BridgeABI.abi);
+
+    // FIXME replace with useCalls
```

Comment:
this is already done ?

Reply:
Nope. It's about replacing the multiple separate `useCall` and `useToken` and `useTokenBalance` with a single `useCalls` for performance: https://usedapp-docs.netlify.app/docs/api%20reference/hooks/#usecalls